web-registry/src/lib/db/types/json-ld/credit-class-metadata.ts

```diff
@@ -24,4 +24,22 @@ export interface CreditClassMetadataLD {
   'regen:carbonOffsetStandard'?: CompactedNameUrl;
   'regen:tokenizationSource'?: string;
   'regen:certifications'?: Certification[];
+  'regen:coBenefits'?: string[];
+  'regen:measuredGHGs': string[];
+  'regen:bufferPoolAccounts': {
+    // We probably want to simplify this to be just a @list instead,
+    // but keeping as is, until C04 class metadata is updated accordingly.
+    '@type': 'schema:ItemList';
+    'schema:itemListElement': BufferPoolAccount[];
+  };
 }
+
+export type BufferPoolAccount = {
+  'schema:name': string;
+  // Keeping both regen:walletAddress and regen:address for now,
+  // until C04 class metadata gets fixed.
+  // But ultimately, we should just use regen:address.
```

Comment:
Also relevant to https://github.com/regen-network/regen-registry-standards/pull/57